src/Analyzer.Core/Rules/BuiltInRules.json

```diff
@@ -663,5 +663,49 @@
         }
       ]
     }
+  },
+  {
+    "name": "SQL_Retention_Days",
+    "description": "SQL servers with auditing to storage account destination should be configured with 90 days retention or higher",
+    "recommendation": "Set the data retention for your SQL Server's auditing to storage account destination to at least 90 days",
+    "helpUri": "https://github.com/Azure/template-analyzer/docs/built-in-bpa-rules.md/#sql-servers-with-auditing-to-storage-account-destination-should-be-configured-with-90-days-retention-or-higher",
+    "evaluation": {
+      "resourceType": "Microsoft.Sql/servers",
+      "where": {
+        "path": "properties.kind",
+        "regex": "^((?!analytics).)*$"
+      },
+      "allOf": [
+        {
+          "resourceType": "Microsoft.Sql/servers/auditingSettings",
+          "where": {
+            "path": "name",
+            "equals": "default"
+          },
+          "anyOf": [
+            {
+              "allOf": [
+                {
+                  "path": "properties.isAzureMonitorTargetEnabled",
+                  "equals": true
+                },
+                {
+                  "path": "properties.storageEndpoint",
+                  "equals": ""
```

Comment:
This might need to be changed to `"hasValue": false`.  The policy is looking for an empty string, but if a template is deployed without this field specified, it probably ends up being an empty string when the policy is evaluated.  We should account for that in the rule.